src/cli.ts

```diff
@@ -2,39 +2,34 @@ import {writeFileSync} from 'fs'
 import path from 'path'
 import {homedir} from 'os'
 import * as yargs from 'yargs'
+import * as Types from './types'
 
 import {ts} from 'ts-morph'
 
 import extractFunctions from './index'
 
 // .option('directory', {type: 'positional'})
 
-const cli = yargs
-  .option('out', {
-    // TODO: figure out how to show the description on --help
-    type: 'string',
-    description: 'Out file path. If missing will write to stdout',
-    hidden: false,
-  })
+const cli = yargs.option('out', {
+  // TODO: figure out how to show the description on --help
+  type: 'string',
+  description: 'Out file path. If missing will write to stdout',
+  hidden: false,
+})
```

Comment:
Sorry, my editor runs prettier on save. I can disable this if it's annoying. I'd suggest adding `prettier --check` to a step in CI.

Reply:
No, that's fine, should update prettier to the latest version on format everything